examples/preprocessing/plot_scaling_importance.py

```diff
@@ -137,3 +180,42 @@
 plt.tight_layout()
 
 plt.show()
+
+# %%
+# Effect of rescaling on model's accuracy
+# =======================================
+#
+# Here we show prediction accuracies in scaled and unscaled data.
+
+from sklearn.metrics import accuracy_score
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegression
+
+unscaled_clf = make_pipeline(pca, LogisticRegression())
+unscaled_clf.fit(X_train, y_train)
+y_pred = unscaled_clf.predict(X_test)
+
+scaled_clf = make_pipeline(scaler, pca, LogisticRegression())
```

Comment:
Note that the scaling influences the L2 penalty. Those 2 logreg models are not really comparable as the L2 norm of their coefficients is potentially quite different (difference "scale").

The performance numbers would be much different if the penalty were chosen by cross validation.

Reply:
In this case, we could use a `LogisticRegressionCV` in the pipeline to account for this particularity.